src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
```

Comment:
no needs to use a template string
```suggestion
      My name is {name}
```